docs/pages/versions/unversioned/sdk/map-view.mdx

```diff
@@ -150,7 +150,7 @@ const styles = StyleSheet.create({
 <Step label="3">
 #### Add the API key to your project
 
-- Copy your **API Key** into your your to either a **.env** file and then add it to your **app.json** under the `ios.config.googleMapsApiKey` field like or copy it:
+- Copy your **API Key** into your **.env** file and then add it to your **app.json** under the `ios.config.googleMaps.apiKey` field or copy:
```

Comment:
`ios.config.googleMapsApiKey` is the correct property name as per the example below and app.json reference; https://docs.expo.dev/versions/unversioned/config/app/#googlemapsapikey.